src/Sylius/Behat/Context/Ui/Admin/ManagingPaymentMethodsContext.php

```diff
@@ -165,11 +174,12 @@ public function thisPaymentMethodElementShouldBe($element, $value)
     }
 
     /**
-     * @Given I want to create a new payment method
+     * @When I want to create a new offline payment method
+     * @When I want to create a new payment method with :factory gateway factory
      */
-    public function iWantToCreateANewPaymentMethod()
+    public function iWantToCreateANewPaymentMethod($factory = 'Offline')
     {
-        $this->createPage->open();
+        $this->createPage->open(['factory' => array_search($factory, $this->gatewayFactories)]);
```

Comment:
Let's pass `true` as the third argument to perform strict comparison instead of loose one.